docs/spec_draft.md

```diff
@@ -277,6 +280,46 @@ Produces a `result` tensor from a constant `value`.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.cosine
+
+`stablehlo.cosine(operand) -> result`
+
+### Semantics
+
+Performs element-wise cosine operation on `operand` tensor and produces a
+`result` tensor, according to the IEEE-754 specification. Numeric precision is
+implementation-defined.
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `operand` | tensor of floating-point or complex element types |
```

Comment:
Can we say something for the complex numbers? We can check with HLO?

For example for sine,
```
sine(a-+ib) = sin(a)*cosh(b) +i cos(a)*sinh(b)
```
Same for other ops.

https://proofwiki.org/wiki/Sine_of_Complex_Number/Proof_1

Reply:
Ignore the previous comments. 
From the fact that this is a standard textbook information,  let us just skip mentioning it here in the spec. Same for ops below